cmd/argo/commands/submit.go

```diff
@@ -61,6 +71,7 @@ func NewSubmitCommand() *cobra.Command {
 	command.Flags().Int32Var(&priority, "priority", 0, "workflow priority")
 	command.Flags().StringVarP(&submitOpts.ParameterFile, "parameter-file", "f", "", "pass a file containing all input parameters")
 	command.Flags().StringVarP(&submitOpts.Labels, "labels", "l", "", "Comma separated labels to apply to the workflow. Will override previous values.")
+	command.Flags().StringVar(&from, "from", "", "Submit from a WorkflowTemplate or CronWorkflow. E.g., --from=CronWorkflow/hello-world-cwf")
```

Comment:
how about `StringVarP` with `-f` flag like `kubectl apply -f`?

Reply:
`--from` is actually the correct K8s pattern:

```
$  kubectl create job --help
Create a job with the specified name.

 ...

  # Create a job from a CronJob named "a-cronjob"
  kubectl create job test-job --from=cronjob/a-cronjob

Options:
...
      --from='': The name of the resource to create a Job from (only cronjob is supported).
...
```